msal/src/main/java/com/microsoft/identity/client/MultipleAccountPublicClientApplication.java

```diff
@@ -34,6 +34,17 @@
     implements IMultipleAccountPublicClientApplication {
     private static final String TAG = MultipleAccountPublicClientApplication.class.getSimpleName();
 
+    /**
+     * Callback for asynchronous loading of broker AccountRecord accounts (in Multiple account mode).
+     */
+    public interface GetAccountRecordsFromBrokerCallback {
```

Comment:
Can we declare this inside of the `BrokerMsalController` instead so that even though this is `public` it remains inside the `*.internal.*` package space?